client/modules/wj-css/src/base/_base.scss

```diff
@@ -0,0 +1,55 @@
+// Normalize html/body
+html,
+body {
+  min-width: 100%;
+  max-width: 100%;
+  min-height: 100%;
+  font-size: 1rem;
+  line-height: 1.5;
+  touch-action: manipulation;
+  scroll-behavior: smooth;
+}
+
+// Sets text selection color.
+::selection {
+  color: var(--col-text);
+  background-color: var(--col-text-select);
+}
+
+// Horizontal rules.
+hr {
+  display: block;
+  height: 0.125rem;
+  background: var(--col-border);
+}
+
+// Indent lists.
+ul,
+ol {
+  padding-left: 1rem;
+}
+
+// Prevents `<sub>` and `<sup>` elements from affecting line-height.
+sub,
+sup {
+  font-size: 75%;
+  line-height: 0;
+}
+
+// Styles underlines globally.
+* {
+  text-decoration-skip-ink: none;
+  text-underline-offset: 0.075em;
+  text-decoration-thickness: 0.05em;
+}
+
+// Struck-out text.
+s {
+  text-decoration: line-through 0.15em;
+}
```

Comment:
Why can't this be left to the browser? Requesting changes here

Reply:
This underline style is:
1. consistent
2. solid

The latter is more important than you may think - by having the underline styled like this, you can e.g. color the bottom of a span of text as a solid color (critic markup comes to mind). Additionally, the output of this style is closer to what people expect to happen when they underline something (imagine underlining something with a pen).

And if we don't like it - we can change it later. The reason this stylistic bit was added was because the editor benefits from this underlining setup.

Reply:
wait actually I keep getting mixed up by the stupid preview, you mean the bottom CSS rule there - one sec, editing comment

Reply:
Why disable specifically skip-ink, for example?

Reply:
I guess this is a difference in opinion here? I don't really care what the browser default is - honestly, they usually kind of suck. The browser default font is Times New Roman.

About skip-ink: I find that underlines generally look better and "work better" (e.g. styling things with underlined styles, like some types of highlighted text) with skip-ink off.